Library/ViewModels/RewardCardContainerViewModel.swift

```diff
@@ -112,10 +112,12 @@ private func pledgeButtonTitle(project: Project, reward: Reward) -> String? {
   case (.nonBacked(.live), _, true):
     return Strings.Select()
   case (.backed(.nonLive), false, _),
-       (.nonBacked(.nonLive), _, _):
+       (.nonBacked(.nonLive), _, _), (.backed(live: .successful), _, _):
```

Comment:
Nit: I don't think we need the `live:` here. Also, I would find this a little easier to read if it were spread over three lines instead of one line with one statement and one line with two

Reply:
👍 happy to update. I'm not really a fan of using one method to handle all of these different pledge states